lib/lotus/utils/path_prefix.rb

```diff
@@ -82,7 +82,7 @@ def relative_join(strings, separator = @separator)
         prefix = @string.gsub(@separator, separator)
 
         self.class.new(
-          Utils::Kernel.Array([prefix, strings]).join(separator),
+          [prefix, strings].join(separator).chomp("/"),
```

Comment:
@jodosha @AlfonsoUceda  I have personally ran against router, lotus and (obviously) utils. I will run it against the rest of the gems in a bit to ensure.